examples/analog_hamiltonian_simulation/03_Parallel_Jobs_on_Aquila.ipynb

```diff
@@ -0,0 +1,578 @@
+{
```

Comment:
<ol><li>"Each batch consists of an area proportional to the bounding box of the register as well as padding to make sure each batch is sufficiently isolated from one another to prevent entanglement between results via the Rydberg interaction. " -&gt; Let's start with the goal: "To prevent entanglement between batches, we need to place them far apart.</li><li>It's unclear what "scale" refers to in the first equation. Please explain.</li><li>"inter-batch interaction" is not a quantity. Please be exact here: "The ratio between intra- and inter-batch interaction strength is ..." </li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/205/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>